### PR TITLE
[refs #81147] Add edit for prefLabel for concepts

### PR DESCRIPTION
**Pull request by dianaboiangiu**

### CODE REVIEW

File: gemet/thesaurus/edit_views.py [hunk 1]
```diff
@@ -0,0 +1,53 @@
...
+            response = HttpResponse(json.dumps({"errors": form.errors}),
+                                    content_type="application/json")
+            response.status_code = 400
+            return response
+
+        field = Property.objects.filter(language=request.POST['language'],
+                                        concept__code=request.POST['concept'],
+                                        name=request.POST['name'])
+
+        published_field = field.filter(status=1).first()
```
Comment (iuliachiriac) [inline gemet/thesaurus/edit_views.py]: Replace hardcoded value `1` with value imported from `models.py` (`Property.PUBLISHED`).

File: gemet/thesaurus/edit_views.py [hunk 2]
```diff
@@ -0,0 +1,53 @@
...
+        pending_field = field.filter(status=0).first()
+
+        if pending_field:
+            pending_field.value = request.POST['value']
+            pending_field.save()
+            field = pending_field
+
+        else:
+            language = Language.objects.get(code=request.POST['language'])
+            concept = Concept.objects.get(code=request.POST['concept'])
```
Comment (iuliachiriac) [inline gemet/thesaurus/edit_views.py]: `get_or_404()` method OR verify if objects exist, else return 404 response code and json response - something like `{"status": "error", "message": "Object not found"}`

File: gemet/thesaurus/edit_views.py [hunk 3]
```diff
@@ -0,0 +1,53 @@
...
+            # Todo: Remove when version is stable
+            field = Property.objects.create(
+                language=language, concept=concept,
+                name=request.POST['name'], value=request.POST['value'],
+                status=0,
+                is_resource=is_resource,
+                version_added=version)
+
+        return HttpResponse(
+            json.dumps({"data": "success", "status_code": "200",
```
Comment (iuliachiriac) [inline gemet/thesaurus/edit_views.py]: Remove `status_code` from json response.
Replace `data` with `status`.

File: gemet/thesaurus/forms.py [hunk 4]
```diff
@@ -23,3 +23,10 @@ class ExportForm(forms.Form):
         empty_label=None,
         label="Choose the language",
     )
+
+
+class PropertyForm(ModelForm):
+
+    class Meta:
+        model = Property
+        fields = ('concept', 'language', 'name', 'value', 'is_resource')
```
Comment (iuliachiriac) [inline gemet/thesaurus/forms.py]: Remove `is_resource` from fields list.

**Comment by dianaboiangiu** [comment]
Requested changes made.

File: gemet/thesaurus/edit_views.py [hunk 5]
```diff
@@ -0,0 +1,210 @@
...
+            concept_code = Concept.objects.get(id=concept['id']).code
+            concept_rev = reverse('concept', kwargs={
+                'langcode': kwargs['langcode'],
+                'code': concept_code})
+            concept['href'] = remove_rev
+            concept['href_add'] = add_rev
+            concept['href_concept'] = concept_rev
+
+    def get(self, request, **kwargs):
+        relation_type = request.GET['type']
```
Comment (iuliachiriac) [inline gemet/thesaurus/edit_views.py]: Why do you use a GET parameter and not a URL argument for `type`?

**Pull request merged**